server/bundler/loader.js

```diff
@@ -80,50 +99,51 @@ function getRequires( sections ) {
 }
 
 function splitTemplate( path, section ) {
-	var pathRegex = getPathRegex( path ),
+	const pathRegex = getPathRegex( path ),
 		sectionString = JSON.stringify( section ),
 		sectionNameString = JSON.stringify( section.name ),
 		moduleString = JSON.stringify( section.module ),
-		envIdString = JSON.stringify( section.envId ),
-		result;
-
-	result = [
-		'page( ' + pathRegex + ', function( context, next ) {',
-		'	var envId = ' + envIdString + ';',
-		'	if ( envId && envId.indexOf( config( "env_id" ) ) === -1 ) {',
-		'		return next();',
-		'	}',
-		'	if ( _loadedSections[ ' + moduleString + ' ] ) {',
-		'		controller.setSection( ' + sectionString + ' )( context );',
-		'		context.store.dispatch( activateNextLayoutFocus() );',
-		'		return next();',
-		'	}',
-		'	if ( config.isEnabled( "restore-last-location" ) && restoreLastSession( context.path ) ) {',
-		'		return;',
-		'	}',
-		'	context.store.dispatch( { type: "SECTION_SET", isLoading: true } );',
-		'	require.ensure([], function( require ) {',
-		'		context.store.dispatch( { type: "SECTION_SET", isLoading: false } );',
-		'		controller.setSection( ' + sectionString + ' )( context );',
-		'		if ( ! _loadedSections[ ' + moduleString + ' ] ) {',
-		'			require( ' + moduleString + ' )( controller.clientRouter );',
-		'			_loadedSections[ ' + moduleString + ' ] = true;',
-		'		}',
-		'		context.store.dispatch( activateNextLayoutFocus() );',
-		'		next();',
-		'	}, function onError( error ) {',
-		'		if ( ! LoadingError.isRetry() ) {',
-		'			console.warn( error );',
-		'			LoadingError.retry( ' + sectionNameString + ' );',
-		'		} else {',
-		'			console.error( error );',
-		'			context.store.dispatch( { type: "SECTION_SET", isLoading: false } );',
-		'			LoadingError.show( ' + sectionNameString + ' );',
-		'		}',
-		'		return;',
-		'	},',
-		sectionNameString + ' );',
-		'} );\n'
+		envIdString = JSON.stringify( section.envId );
+
+	const result = [
```

Comment:
Could you back out the const result assignment? It's making reading this diff a bit difficult

Reply:
I would recommend changes like that for a separate janitorial PR

Reply:
I'm also in favor of not touching that as part of that PR too much...
We have an ongoing PR of refactoring this already, maybe you can comment/commit [there](https://github.com/Automattic/wp-calypso/pull/18383)?

Reply:
I agree, we should differentiate refactoring PRs from feature/bugfix ones. A refactoring PR should take into account existing efforts to improve the code and there is at least one as Yury mentioned!

@stephanethomas could you undo all changes that aren't needed for that patch?

Reply:
I disagree with this for a few reasons, though of course "it depends" what the right solution is.

- In practice it's unlikely that we will actually follow up with said refactoring, so it's good to go ahead and refresh code that you are touching (as long as the changes are not too severe).
- Without retouching modified lines to conform to current coding standards, we will get lint errors and future changes from `prettier` auto-formatting.

In the case of this PR, reviewing it with the [`?w=1` argument](https://github.com/Automattic/wp-calypso/pull/19396/files?w=1) makes the actual changes really obvious.

Reply:
What James said :).

Amber doesn't have time for refactoring these days so quite frankly it would never happen. I know there is another attempt to refactor this part of the code, but it's a pull request started in September and it's still in progress, with no ETA.

I'm happy to revert [to using `var`](https://github.com/Automattic/wp-calypso/pull/19396/commits/6e902771b54bf997183f6cb2a2e715b3ab2c941c) for declaring variables if the [diff with whitespaces ignored](https://github.com/Automattic/wp-calypso/pull/19396/files?w=1) is not enough. Let me know if that's a blocker. Note I've increased the indentation of the generated code here - something that should be part of this pull request - so the [default diff](https://github.com/Automattic/wp-calypso/pull/19396/files) won't be that different from what you currently see.

Reply:
In general I would agree, a refactor is a nice bonus usually, but in this case I don't think this refactor improves the code much, it's a personal syntax preference change in my opinion (`` vs `[''].join( '\n' )` ).

The work started in https://github.com/Automattic/wp-calypso/pull/18383 however is more than a syntactic change, it refactors the whole file, improves the logic and I'm pretty sure we can have it merged soon.

*TLDR:* I'm not a fan of refactors that only slightly improves the code and make the review process harder.

Reply:
> In practice it's unlikely that we will actually follow up with said refactoring 

😢 Splitting out janitorials makes things faster all around. They're easy to setup, and fast to review. If you don't have time for refactoring please keep these commits out of PRs that change behavior. It's an undue burden on the reviewer as it is much more difficult to read the diff and catch issues.